controllers/fenceagentsremediation_controller.go

```diff
@@ -103,22 +107,31 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 	//TODO: Check that FA is excutable? run cli.IsExecuteable
 
 	r.Log.Info("Create and execute the fence agent", "Fence Agent", far.Spec.Agent, "Node Name", req.Name)
-	faParams, err := buildFenceAgentParams(far)
+	faParams, nodeParams, err := buildFenceAgentParams(far)
 	if err != nil {
 		return emptyResult, err
 	}
 	cmd := append([]string{far.Spec.Agent}, faParams...)
 	// The Fence Agent is excutable and the parameters are valid but we don't know about their values
-	if _, _, err := r.Executor.Execute(pod, cmd); err != nil {
-		//TODO: better seperation between errors from wrong shared parameters values and wrong node parameters values
+	outputRes, outputErr, err := r.Executor.Execute(pod, nodeParams, cmd)
+	// Check if not error returned and a successful message was returned
+	if err != nil {
+		//TODO: better separation between errors from wrong shared parameters values and wrong node parameters values
+		if outputErr != "" {
+			err = fmt.Errorf(outputErr+"- %w", err)
+		}
 		return emptyResult, err
 	}
+	if outputRes != successFAMessages {
+		errMes := fmt.Sprintf("unknown fence agent response - expecting a \"%s\" response", successFAMessages)
```

Comment:
Nit: might be useful to report the actual response that was received (i.e `outputRes`)